src/sas/sascalc/data_util/calcthread.py

```diff
@@ -269,7 +270,9 @@ def _run(self):
             except KeyboardInterrupt:
                 pass
             except:
-                self.exception()
+                errors += 1
+        if errors > 0:
+            self.exception()
```

Comment:
This is no longer really an issue. I am going to revert my changes back to the original. This method only runs a single time and many of the errors I am seeing (cannot plot negative numbers on log axis) are coming from the plotting perspective.